src/database/firebase_list_observable.spec.ts

```diff
@@ -51,6 +49,16 @@ describe('FirebaseObservable', () => {
     expect(O.map(noop) instanceof FirebaseListObservable).toBe(true);
   });
 
+  describe('$ref', () => {
+    // it('should be a firebase.database.Reference', () => {
```

Comment:
Can't use `xit` because neither `database.Reference` nor `firebase.database.Reference` will compile in typescript until the bug is fixed : (